<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(47, 79, 79);
font: 75 9pt &quot;Arial&quot;;
</string>
  </property>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string>Connect to device</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayPeripheralButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string>Display peripherals</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeAxisLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>590</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   Time (secs)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="debugTextDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="deviceListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 85);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vCOMRadioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Virtual COM</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bluetoothRadioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Bluetooth</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Peripherals</string>
   </property>
   <widget class="QListView" name="peripheralListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 85);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayDevicesButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string>Display devices</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="openPortButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string>Open port</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="minimumValText">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>320</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>170</y>
     <width>131</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QLabel" name="maxValLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Maximum Value</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="maximumValueText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="minValLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Minimum Value</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sampleRateText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sampleRateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sample Rate</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="deviceDescpButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string>Device Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startLogButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string>Start Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopLogButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string>Stop Log</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="plotterGraphicsView">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>750</width>
     <height>525</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 136, 153);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsViewXAxis">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>554</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsViewYAxis">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>25</width>
     <height>525</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="peripheralSignalListView">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>410</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 85);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="currentSignalOnPlot1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="currentSignalOnPlot2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>580</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>connectButton</zorder>
  <zorder>displayPeripheralButton</zorder>
  <zorder>disconnectButton</zorder>
  <zorder>timeAxisLabel</zorder>
  <zorder>debugTextDisplay</zorder>
  <zorder>deviceListView</zorder>
  <zorder>vCOMRadioButton</zorder>
  <zorder>bluetoothRadioButton</zorder>
  <zorder>openPortButton</zorder>
  <zorder>minimumValText</zorder>
  <zorder>deviceDescpButton</zorder>
  <zorder>startLogButton</zorder>
  <zorder>stopLogButton</zorder>
  <zorder>plotterGraphicsView</zorder>
  <zorder>graphicsViewXAxis</zorder>
  <zorder>graphicsViewYAxis</zorder>
  <zorder>peripheralSignalListView</zorder>
  <zorder>currentSignalOnPlot1</zorder>
  <zorder>currentSignalOnPlot2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>966</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDisconnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>935</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayDevicesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDisplayDevicesButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openPortButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOpenPortButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1015</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayPeripheralButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDisplayPeripheralsButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deviceDescpButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDeviceDescrpButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>995</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>1050</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startLogButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStartLogButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopLogButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStopLogButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onDisconnectButtonClicked()</slot>
  <slot>onDisplayPeripheralsButtonClicked()</slot>
  <slot>onDisplayDevicesButtonClicked()</slot>
  <slot>onOpenPortButtonClicked()</slot>
  <slot>onDeviceDescrpButtonClicked()</slot>
  <slot>onStartLogButtonClicked()</slot>
  <slot>onStopLogButtonClicked()</slot>
 </slots>
</ui>
